scss/_root.scss

```
@@ -13,4 +17,22 @@
   --#{$variable-prefix}font-sans-serif: #{inspect($font-family-sans-serif)};
   --#{$variable-prefix}font-monospace: #{inspect($font-family-monospace)};
   --#{$variable-prefix}gradient: #{$gradient};
+
+  // Root and body
+  --#{$variable-prefix}root-font-size: #{$font-size-root};
+  --#{$variable-prefix}body-font: #{$font-family-base};
+  --#{$variable-prefix}body-font-size: #{$font-size-base};
+  --#{$variable-prefix}body-color: #{$body-color};
+  --#{$variable-prefix}body-bg: #{$body-bg};
+
+  // Layout
+  --#{$variable-prefix}grid-columns: #{$grid-columns};
+  --#{$variable-prefix}grid-gutter-width: #{$grid-gutter-width};
+
+  // Spacing
+  // TODO: Ideally we'd size these values based on the CSS var instead of the Sass map, so they'd be live updating, but TBD on feasibility
+  --#{$variable-prefix}spacer: #{$spacer};
+  @each $size, $value in $spacers {
+    --#{$variable-prefix}spacer-#{$size}: #{$value};
+  }
```

Because users can put arbitrary values in `$spacers`, there may not be a definitive relationship between those values and `$spacer` that can be mimicked by the CSS variables.  But, it may still be useful to scale the variables based on their ratio:

```suggestion
  @each $size, $value in $spacers {
    --#{$variable-prefix}spacer-#{$size}: calc(var(--#{$variable-prefix}spacer) * #{$value / $spacer});
  }
```